### PR TITLE
Fixes #11596: UserManagement technique does not allow to change the gid of an existing user

### PR DESCRIPTION
**Pull request by ncharles**
https://www.rudder-project.org/redmine/issues/11596

### CODE REVIEW

File: techniques/systemSettings/userManagement/userManagement/7.1/metadata.xml [hunk 1]
```diff
@@ -21,6 +21,7 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 It is intended to check the user parameters on the target host.
 -->
 <TECHNIQUE name="User management">
+  <DEPRECATED>This technique version has been superseded by a new version. It will no longer be available in the next stable version of Rudder. Please upgrade to the latest version.</DEPRECATED>
```
Comment (amousset) [inline techniques/systemSettings/userManagement/userManagement/7.1/metadata.xml]: This should not be deprecated for now.

File: techniques/systemSettings/userManagement/userManagement/7.2/metadata.xml [hunk 2]
```diff
@@ -0,0 +1,207 @@
+<!--
+Copyright 2016 Normation SAS
```
Comment (amousset) [inline techniques/systemSettings/userManagement/userManagement/7.2/metadata.xml]: 2017

File: techniques/systemSettings/userManagement/userManagement/7.2/metadata.xml [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+-->
+
+<!--
```
Comment (amousset) [inline techniques/systemSettings/userManagement/userManagement/7.2/metadata.xml]: This comment has been remove in the newer branches and is unnecessary/outdated.

**Comment by ncharles** [comment]
OK, merging this PR

**Comment by ncharles** [comment]
OK, merging this PR

**Pull request merged**